GetDKAN/dkan2
# Add custom node content view

- Adds vbo module
- Adds admin_toolbar
- Adds new view to override default `admin/content` view
- Adds column for data type
- Adds bulk delete operation

## QA steps
- `drush dkan-dummy-content:create`
- disable default drupal admin/content view
- confirm `admin/content` displays new view with vbo bulk delete op and column for data type

![content_view](https://user-images.githubusercontent.com/314172/73554323-21c65400-4411-11ea-8149-47e4fc3d48bc.png)
